yaksh/test_models.py

```diff
@@ -839,6 +859,66 @@ def test_validate_and_regrade_mcc_correct_answer(self):
         self.assertEqual(self.answer.marks, 0)
         self.assertFalse(self.answer.correct)
 
+    def test_validate_and_regrade_code_correct_answer(self):
+        # Given
+        # Start code server
+        
+        user_answer = dedent("""\
+                                def add(a,b):
+                                    return a+b
+                             """)
+        self.answer = Answer(question=self.question1,
+                             answer=user_answer,
+                             )
+        self.answer.save()
+        self.answerpaper.answers.add(self.answer)
+        user = self.answerpaper.user
+
+        # When
+        json_data = self.question1.consolidate_answer_data(user_answer,
+                                                          user
+                                                          )
+        check_result = self.answerpaper.validate_answer(user_answer,
+                                                        self.question1,
+                                                        json_data,
+                                                        self.answer.id,
+                                                        self.SERVER_POOL_PORT
+                                                        )
+        url = 'http://localhost:%s' % self.SERVER_POOL_PORT
+        uid = check_result['uid']
+        while check_result['status'] != "done":
```

Comment:
Should we have a counter for this?
I mean how many times max it should request for the result.

Space after url comma